sdk/tables/Azure.Data.Tables/src/TableRestClient.cs

```diff
@@ -166,7 +166,7 @@ public Response<List<Response>> SendBatchRequest(HttpMessage message, List<(ITab
 
         private static string AppendEntityInfoToMessage(string messsage, ITableEntity entity)
         {
-            return messsage += $"\n\nRowKey={entity.RowKey}";
+            return messsage += $"\nRowKey={entity.RowKey}\n";
```

Comment:
Do we typically have a new line at the end of the exception message?

Reply:
Not sure - I added it while testing when I saw how it blended with the next line when the test initially failed. Do you think it should be removed?

Reply:
removed it and replaced with a trailing `.`